source/includes/job-board/_applications.md

```diff
@@ -217,3 +238,26 @@ longitude for the selected location using the [Google Place Details API](https:/
 this API call.
 
 Note that all 3 fields must be included. If only `location` is sent and `latitude` and `longitude` are omitted, `location` will be ignored entirely.
+
+### Validations on POST Requests
+
+**Job validations:**
+
+* The job must be live and published
+* The job board must be live and published
+* The job post ID must exist on the job board with the supplied board_token
+
+**Field validations:** 
+
+* `first_name`, `last_name`, and `phone` fields must not contain a URL
+* `email` field must contain a valid email address
+* `educations` object must contain valid `school_name_id`, `degree_id`, `discipline_id`, `start_date`, and `end_date`
+* `employments` object must contain `company_name`, `title`, `current`, `start_date` and `end_date`
```

Comment:
while i think its fine to leave as is, technically if `current = true`, you dont need an end_date.

And the opposite is also true. if there is no `current` or `current = false`, then there needs to be an end_date. this is more FYI instead of us documenting it.